syntax = "proto3";
package locales;


message Locales {
  int32 id = 1;
  string name = 2;
  string code = 3;
  bool is_main = 4;
}


